src/buttons/CopyToClipboard/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'preact/hooks'
+import PropTypes from 'prop-types'
+import CSS from './styles.css'
+
+import copyToClipboard from './../../utils/copy-to-clipboard'
+
+const CopyToClipboard = ({
+	dataTestId,
+	valueToCopy,
```

Comment:
(natural naming) similar comment here, please invert the naming for most of these arguments (and update all consumers of this function, `defaultProps`, and `propTypes` to match) so that they are in a natural English order, like so: 

```js
const CopyToClipboard = ({
	dataTestId,
	toCopyClass,
	toCopyLabel,
	toCopyValue,
	copiedClass,
	copiedLabel,
	copyingClass,
	copyingLabel,
	copyingFeedbackTime,
	copiedFeedbackTime,
}) => {
```